staging/src/k8s.io/kubectl/pkg/cmd/config/view.go

```diff
@@ -44,8 +44,7 @@ type ViewOptions struct {
 	Minify       bool
 	RawByteData  bool
 
-	Context      string
-	OutputFormat string
+	Context string
```

Comment:
I would recommend keeping the same indentation space like the previous variable

Reply:
This is the result of running `go fmt`.